js/charts/heatmap.js

```diff
@@ -517,21 +517,40 @@ function heatmap() {
             .data(yElement)
             .enter().append('text')
             .attr('class', 'dimLabel')
-            .text(function (d) { return d; })
-            .text(function (d) {
-                if (d.length > 4) {
-                    return d.substring(0, 4) + '...';
-                }
-                return d;
-            })
-            .attr('x', 0)
             .attr('y', function (d, i) { return i * cellHeight; })
             .style('fill', dimLabelColor)
             .style('font-style', fontStyleForDimension)
             .style('font-weight', fontWeightForDimension)
             .style('font-size', fontSizeForDimension)
             .style('text-anchor', 'end')
             .attr('transform', 'translate(' + -offset + ',' + cellHeight / 1.75 + ')')
+            .append("tspan")
+            .text(function (d) { return d; })
+            .text(function (d) {
+                if (d.length > 15) {
+                    return d.substring(0, 15);
```

Comment:
15 const is repeating.. Better to place in const file

Reply:
okay